lib/__tests__/disableRanges.test.mjs

```diff
@@ -202,22 +203,58 @@ it('disable single line all rules', async () => {
 });
 
 it('disable single line all rules, in a rule node', async () => {
-	const result = await testDisableRanges(
-		`a, /* stylelint-disable-line */
-		b /* stylelint-disable-line */ {} `,
-	);
+	const result = await testDisableRanges(stripIndent`
+		/* offset by one line */
+		a,/* stylelint-disable-line */
+		b/* stylelint-disable-line */,
+		c,/* stylelint-disable-line */
+		long-selector/* stylelint-disable-line */,
```

Comment:
nit
suggestion

Type selectors don't contain hyphens.
Maybe rename to .long-selector?

Reply:
Could be a custom element, this is perfectly valid CSS/HTML :)